_content/isoman-page.json

```
@@ -0,0 +1,50 @@
+{
+  "isoman_contents": [
+    {
+      "category": "Pannduan Isoman",
+      "title": "Isolasi Mandiri : Apa yang harus dilakukan?",
```

```suggestion
      "title": "Isolasi Mandiri: Apa yang harus dilakukan?",
```

Nitpick: use proper spacing for `:`.

---

sorry, fixed.